hack/windows/Prepare-Node.ps1

```diff
@@ -10,28 +10,38 @@ script from the sig-windows-tools repo release page: https://github.com/kubernet
 - Registers wins as a service in order to run kube-proxy and antrea-agent as DaemonSets.
 - Registers kubelet as an nssm service. More info on nssm: https://nssm.cc/
 - Adds node ip in kubelet startup params.
-- Create virtual netadapter for kube-proxy.
+- (Optional) Create virtual netadapter for kube-proxy.
 - (Optional) Installs OVS kerner driver, userspace binaries. Registers ovsdb-server and ovs-vswitchd services.
 
 .PARAMETER KubernetesVersion
 Kubernetes version to download and use
 
+.PARAMETER InstallKubeProxy
+Install Kube-Proxy
+
 .PARAMETER InstallOVS
 Install OVS
 
 .PARAMETER NodeIP
 The node ip used by kubelet
 
+.PARAMETER ContainerRuntime
+Container that Kubernetes will use. (docker or containerd)
+
 .EXAMPLE
-PS> .\Prepare-Node.ps1 -KubernetesVersion v1.18.0 -InstallOVS -NodeIP 192.168.1.10
+PS> .\Prepare-Node.ps1 -KubernetesVersion v1.27.0 -NodeIP 192.168.1.10 -ContainerRuntime containerd 
 
 #>
 
+
+
 Param(
     [parameter(Mandatory = $false, HelpMessage="Kubernetes version to use")] [string] $KubernetesVersion="v1.18.0",
     [parameter(Mandatory = $true, HelpMessage="Node IP")] [string] $NodeIP,
+    [parameter(Mandatory = $false)] [switch] $InstallKubeProxy = $true,
```

Comment:
I would prefer to use default value as false for parameter `$InstallKubeProxy` since AntreaProxy is enabled by default now.